tests/test_polyhedron.py

```diff
@@ -339,6 +339,96 @@ def test___repr__():
     repr(icosidodecahedron)
 
 
+@combine_marks(
+    named_platonic_mark,
+    named_archimedean_mark,
+    named_catalan_mark,
+    named_johnson_mark,
+    named_prismantiprism_mark,
+    named_pyramiddipyramid_mark,
+)
+def test_edges(poly):
+    # Check that the first column is in ascending order.
+    assert np.all(np.diff(poly.edges[:, 0]) >= 0)
+
+    # Check that all items in the first column are greater than those in the second.
+    assert np.all(np.diff(poly.edges, axis=1) >= 0)
```

Comment:
i-i edges should be illegal so this can be a strict inequality.
```suggestion
    assert np.all(np.diff(poly.edges, axis=1) > 0)
```